src/main/java/com/bazel_diff/VersionProvider.java

```diff
@@ -5,7 +5,7 @@
 class VersionProvider implements IVersionProvider {
     public String[] getVersion() throws Exception {
         return new String[] {
-            "3.2.2"
+            "3.4.0"
```

Comment:
Latest release is `3.4.0` https://github.com/Tinder/bazel-diff/releases/tag/3.4.0